sdcm/cluster_aws.py

```diff
@@ -270,15 +270,15 @@ def update_bootstrap(ec2_user_data, enable_auto_bootstrap):
                 ec2_user_data += ' --bootstrap false '
         return ec2_user_data
 
-    # This is workaround for issue #5179: IPv6 - routing in scylla AMI isn't configured properly
+    # This is workaround for issue #5179: IPv6 - routing in scylla AMI isn't configured properly (when not Amazon Linux)
     @staticmethod
     def network_config_ipv6_workaround_script():  # pylint: disable=invalid-name
         return dedent("""
             BASE_EC2_NETWORK_URL=http://169.254.169.254/latest/meta-data/network/interfaces/macs/
             MAC=`curl -s ${BASE_EC2_NETWORK_URL}`
             IPv6_CIDR=`curl -s ${BASE_EC2_NETWORK_URL}${MAC}/subnet-ipv6-cidr-blocks`
 
-            sudo ip route add $IPv6_CIDR dev eth0
+            grep -qi "amazon linux" /etc/os-release || sudo ip route add $IPv6_CIDR dev eth0
```

Comment:
why do you need this grep?
the if "not node.distro.is_amazon2" should be enough.

Reply:
because this script is sent to the node creation, and by the time it is executed, i had no way to know what OS it is running, so, this grep is to know if we need to run the workaround or not (on Amazon Linux 2 we don't need to run it)

Reply:
it will return `1` if it is not Amazon Linux 2 and then run the `sudo ip route...`, otherwise, it doesn't run it if `amazon linux` is found in the `/etc/os-release`